pkg/internal/session/session.go

```diff
@@ -177,7 +177,7 @@ func (h *handler) waitForRefToComplete() (*istiov1alpha1.Session, string, error)
 	if h.opts.Duration != nil {
 		duration = *h.opts.Duration
 	}
-	err = wait.Poll(duration, 10*time.Second, func() (bool, error) {
+	err = wait.Poll(duration, 60*time.Second, func() (bool, error) {
```

Comment:
microk8s slower?

Reply:
Yeah, it looked like that yesterday. Was wondering the same. Maybe I will bring it back to 10.

```suggestion
	err = wait.Poll(duration, 10*time.Second, func() (bool, error) {
```